.travis.yml

```diff
@@ -21,5 +21,5 @@ script:
   - sh -x ./node_modules/patternfly-eng-release/scripts/_build.sh -o
 
 after_success:
-  - grunt build
+  - grunt build:production
```

Comment:
The travis scripts are executed sequentially, so it would look like:
- <build script>
- <release script>
- <publish script>

These single purpose scripts more clearly indicate what travis is accomplishing.